source/_integrations/discovery.markdown

```diff
@@ -107,12 +107,27 @@ Valid values for enable are:
 Home Assistant must be on the same network as the devices for UPnP discovery to work.
 If running Home Assistant in a [Docker container](/docs/installation/docker/) use switch `--net=host` to put it on the host's network.
 
+If for some reason you do not want to use `--net=host` you can also configure avahi-reflector on your host to make mDNS / UPnP work:
```

Comment:
The section should be renamed to `mDNS / UPnP`.
However, I think avahi-reflector only solves mDNS, not UPnP?